HACKTOBERFEST.md

```diff
@@ -0,0 +1,27 @@
+# :pizza: Hacktoberfest 2019 :beer:
+
+It's that time of the year again!
+[Hacktoberfest](https://hacktoberfest.digitalocean.com) is here, and `gensim` needs **your** help.
+We've prepared a list of good issues to work on [here](https://github.com/RaRe-Technologies/gensim/labels/hacktoberfest).
+
+If the learning curve for `gensim` is a bit steep, give the [smart_open](https://github.com/RaRe-Technologies/smart_open) repository a try.
+`smart_open` is an important dependency of `gensim`: it performs file I/O over a variety of protocols and formats.
+There's also a list of Hacktoberfest-friendly issues to work on [here](https://github.com/RaRe-Technologies/smart_open/labels/hacktoberfest).
+
+Of course, we welcome contributions on any of the existing issues, not just the ones labeled `hacktoberfest`.
+Just be sure that no-one else is already working on it first.
+
+Furthermore, we also welcome contributions not connected to an existing issue.
+This includes things like fixing typos in documentation, docstrings, etc.
+If you make such contributions, please make the motivation behind the contribution clear.
+Please avoid making innocuous changes without sufficient motivation (e.g. changing code formatting, etc).
+
+## Before Contributing
+
+Check out the following:
+
+- [First-time contributors guide](https://github.com/firstcontributions/first-contributions): if this is your first time contributing on GitHub.
+- [Hacktoberfest rules](https://hacktoberfest.digitalocean.com/faq#rules): read this in full
+- [Developer page](https://github.com/RaRe-Technologies/gensim/wiki/Developer-page) on our Wiki: for the git flow, code style, etc.
+
```

Comment:
add links to the person that contributor can ask about anything (if any questions)

Reply:
OK, linked to my own Telegram and Twitter